src/components/formComponentTemplateTxnOps.tsx

```diff
@@ -84,17 +96,24 @@ export const formComponentTemplateTxnOps = ({
         validate: validate.amount,
       };
     case "asset":
+    case "send_asset":
+    case "dest_asset":
       return {
         render: (templ: TemplateRenderAssetProps) => (
           <AssetPicker
             key={id}
             assetInput="alphanumeric"
             id={id}
-            label="Asset"
+            label={custom?.label ?? "Asset"}
             labelSuffix={!templ.isRequired ? "optional" : undefined}
             value={assetPickerValue(templ.value)}
             error={templ.error}
-            includeNative
+            note={custom?.note}
+            includeNative={
+              typeof custom?.includeNative === "undefined"
```

Comment:
if `includeNative` is `undefined`, native is included?

Reply:
Yes, by default, it's enabled because we show it more often than hide it. If it's set to `false` in custom config, then use that value to hide the native asset option.